### ISSUE TITLE
after closeChangeStream runs, search is not working 

### ISSUE THREAD

**Issue by BekaArabidze98** [typesense/typesense#965]
```
`const closeChangeStream = ({ timeInMs = 2000, changeStream }) => {
     return new Promise((resolve) => {
         setTimeout(() => {
             changeStream.close();
             console.log('closing change stream');
             resolve();
         }, timeInMs);
     });
 };`
```

After two seconds when closeChangeStream functions runs, I can not search anymore. 
I mean It does not search anymore.Without this function everything works perfectly.
```
const monitorListingsUsingEventEmitter = async ({
    changeStream,
    // timeInMs = 8000,
    schema,
    excludeFields,
}) => {
    try {
        changeStream.on('change', (next) => {
            index({ next, schema, excludeFields });
        });
        // await closeChangeStream({ timeInMs, changeStream });
    } catch (err) {
        console.log('error', err);
    }
};
```

I believe when a user starts searching change stream should open by It self,right?

**Comment by jasonbosco** [comment]
I can't see any Typesense related things in the code snippet you shared... Could you rephrase?

**Comment by BekaArabidze98** [comment]
oh, sorry. I just saying that after closeChangeStream  function runs I cant search anymore. Thats why I have commented out 
closeChangeStream function in monitorListingsUsingEventEmitter  function. 
I was just wondering, If anyone had this kind of case before.

Long story shors, when the stream closes, I cant search anymore.
